test/gtest/common/test_helpers.cc

```diff
@@ -685,6 +685,18 @@ const struct sockaddr* sock_addr_storage::get_sock_addr_ptr() const {
     return m_is_valid ? (struct sockaddr *)(&m_storage) : NULL;
 }
 
+const void* sock_addr_storage::get_sock_addr_in_buf() const {
+    const struct sockaddr* saddr = get_sock_addr_ptr();
+
+    EXPECT_NE((const struct sockaddr*)NULL, saddr);
```

Comment:
ucs_assert_always

Reply:
actually there is a dereferencing on next line, but ok will change as well